### PR TITLE
[bug 1137022] Minor settings cleanup

### PR DESCRIPTION
**Pull request by willkg**
* remove some more settings that aren't used anywhere (DOMAIN, PROTOCOL,
  PORT)
* move NOSE_ARGS settings from local.py-dist to test.py where they
  really should be
* (cosmetic) fix a typo in requirements.txt
* remove DEBUG = True from test.py since Django turns it False for
  tests anyhow

Quick r?

### CODE REVIEW

**Comment by willkg** [comment]
I'm trying to break up this massive change to settings stuff into a few smaller PRs so it's easier to review. This one is mostly cosmetic.

**Comment by willkg** [comment]
Ugh. The settings are so messed up. Re-added the `DEBUG = True` to the `travis.py-dist` file so the tests pass with travis again. I'll nix it in the big PR later after I've fixed `validate_settings`.

File: requirements/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Requirements from github urls because we're using non-reelased things
+# Requirements from github urls because we're using non-released things
```
Comment (rehandalal) [inline requirements/requirements.txt]: i prefer reelased.

**Comment by rehandalal** [comment]
lgtm. travis is happy. :ship:

**Comment by willkg** [comment]
99c84d3 [bug 1137022] Minor settings cleanup

Thank you!

**Pull request merged**